docs/topics/settings.rst

```diff
@@ -81,6 +81,22 @@ do so by setting their :attr:`~scrapy.Spider.custom_settings` attribute:
             "SOME_SETTING": "some value",
         }
 
+It's often better to provide a :meth:`~scrapy.Spider.update_settings` instead,
+and settings set there should use the "spider" priority explicitly:
+
+.. code-block:: python
+
+    import scrapy
+
+
+    class MySpider(scrapy.Spider):
+        name = "myspider"
+
+        @classmethod
+        def update_settings(cls, settings):
+            settings.set("SOME_SETTING", "some value", priority="spider")
+            super().update_settings(settings)
```

Comment:
No strong opinion, since `priority` will handle most cases, but I do wonder if reversing these lines may be a better default suggestion, even if for something as trivial as keeping the `super()` call closer to the method definition start, to make it more clear that it is being called in cases where lots of settings are defined (i.e. lots of lines before `super` if we leave `super` for last).

```suggestion
            super().update_settings(settings)
            settings.set("SOME_SETTING", "some value", priority="spider")
```

Which also applies to the later example, where to a [similar suggestion](https://github.com/scrapy/scrapy/pull/5846/files#r1230629675) I responded “does not matter” :sweat_smile: .